app/src/main/java/com/yeonae/chamelezone/view/login/JoinFragment.kt

```diff
@@ -12,33 +12,35 @@ import android.widget.Toast
 import androidx.fragment.app.Fragment
 import com.yeonae.chamelezone.Injection
 import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.network.model.EmailResponse
+import com.yeonae.chamelezone.network.model.NicknameResponse
 import com.yeonae.chamelezone.view.login.presenter.JoinContract
 import com.yeonae.chamelezone.view.login.presenter.JoinPresenter
 import kotlinx.android.synthetic.main.fragment_join.*
 import java.util.regex.Pattern
 
 class JoinFragment : Fragment(), JoinContract.View {
-    var checkedEmail: Boolean = false
-    var checkedNickname: Boolean = false
-    override fun showNicknameMessage(response: Boolean) {
-        if(response){
+    private var checkedEmail: Boolean = false
+    private var checkedNickname: Boolean = false
+    override fun showNicknameMessage(response: NicknameResponse) {
```

Comment:
```suggestion
    override fun showNicknameMessage(nickNameCheck: Boolean) {
```
view 에서는 필요한 정보만 받아서 사용하는게 좋을 것 같아요